modules/auth/proc-adding-google-as-an-authentication-provider.adoc

```diff
@@ -0,0 +1,59 @@
+[id='proc-adding-google-as-an-authentication-provider_{context}']
+= Enabling the Google authentication provider
+
+{product} includes a Google authentication provider that can authenticate users by using Google OAuth2.
+
+.Procedure
+. To allow {product-short} to authenticate with Google, create OAuth credentials in Google.
+.. Log in to the link:https://console.cloud.google.com/[Google Console].
+.. Select or create a new project from the dropdown menu on the top bar.
+.. Go to link:https://console.cloud.google.com/apis/credentials/consent[*APIs & Services > OAuth consent screen*], and click *Edit app* to configure the consent screen:
+**Authorized domains**::
+Enter your {product-short} domain (_<APP_FQDN>_).
+*Scopes*:: Add these scopes:
+`openid`::: Associate you with your personal info on Google.
+`auth/userinfo.email`::: See your primary Google Account email address.
+`auth/userinfo.profile`::: See your personal info, including any personal info you've made publicly available.
+*User type*:: Add yourself as a test user, if using the `External user` type.
+.. Go to to link:https://console.cloud.google.com/apis/credentials[*APIs & Services > Credentials*], click *Create Credentials*, choose *OAuth client ID*, and create a credential with following parameters:
+*Application type*:: Select `Web Application`.
+*Name*:: Enter your application name, such as `{product-short}`.
+*Authorized JavaScript origins*:: Add URI: `pass:c,a,q[_<APP_FQDN>_]`
+*Authorized redirect URIs*:: Click *Add URI* to enter the backend authentication URI set in {product-short}:
+`pass:c,a,q[_<APP_FQDN>_/api/auth/google/handler/frame]`
+.. Save the *Client ID* and *Client secret* for the next step.
+
+. Add your Google credentials in your {product-short} secrets.
++
+Edit your {product-short} secrets, such as `secrets-rhdh`.
++
+Add the following key/value pairs:
++
+`AUTH_GOOGLE_CLIENT_ID`:: Enter the *Client ID* that you generated on Google.
+`AUTH_GOOGLE_CLIENT_SECRET`:: Enter the *Client secret* that you generated on Google.
+
```

Comment:
Numbers are missing from these steps. If they are sub-steps, format them as such.

Reply:
I mean: intentionally unordered.

Reply:
So _Edit your {product-short} secrets, such as `secrets-rhdh`_ and _Add the following key/value pairs_ are not procedural steps? What is being defined there?